Examples fail to run on master with a 'buble-loader' error

Having a problem running the examples on the master branch.

`npm install` and `npm test` both succeed (or appear to, still a bit of a npm newbie.

But when I run `npm start`, I get the error

```
ERROR in multi (webpack)-dev-server/client?http://localhost:8082 webpack/hot/dev-server ./app.js
Module not found: Error: Can't resolve 'buble-loader' in '/Users/chris/code/react-map-gl/examples/custom-interactions'
 @ multi (webpack)-dev-server/client?http://localhost:8082 webpack/hot/dev-server ./app.js
```

Happy to provide any other debug information as needed!

--- a/package.json
+++ b/package.json
@@ -38,7 +38,6 @@
     "bowser": "^1.2.0",
     "immutable": "*",
     "mapbox-gl": "0.32.1",
-    "path-exists": "^3.0.0",
     "viewport-mercator-project": "^4.0.0-alpha.5"
   },
   "devDependencies": {
@@ -56,6 +55,7 @@
     "glob": "^7.1.1",
     "jsdom": "~9.9.1",
     "module-alias": "^2.0.0",
+    "path-exists": "^3.0.0",
     "pre-commit": "^1.2.2",
     "react": "^15.4.0",
     "react-addons-test-utils": "^15.4.0",

--- a/package.json
+++ b/package.json
@@ -22,11 +22,10 @@
     "postinstall.js"
   ],
   "scripts": {
-    "start": "(cd examples/custom-interactions && npm run start-local)",
+    "start": "(cd examples/custom-interactions && (path-exists node_modules || npm i) && npm run start-local)",
     "build": "npm run clean && npm run build-es5 && npm run build-es6",
     "build-es5": "mkdir -p dist && babel src --out-dir dist --plugins=transform-es2015-modules-commonjs --source-maps inline",
     "build-es6": "mkdir -p dist-es6 && babel src --out-dir dist-es6 --source-maps inline",
-    "clean": "rm -fr dist/* dist-es6/*",
     "clean": "rm -fr dist/* dist-es6/*",
     "lint": "eslint src example test",
     "test": "npm run lint && node test/node.js",
@@ -39,6 +38,7 @@
     "bowser": "^1.2.0",
     "immutable": "*",
     "mapbox-gl": "0.32.1",
+    "path-exists": "^3.0.0",
     "viewport-mercator-project": "^4.0.0-alpha.5"
   },
   "devDependencies": {

